src/Package.php

```diff
@@ -163,9 +165,9 @@ public function hasMigrations(...$migrationFileNames): static
         return $this;
     }
 
-    public function hasCommand(string $commandClassName): static
+    public function hasCommand(string $commandClassName, bool $justConsole = false): static
```

Comment:
Is this `flag` being used somewhere? Seems like the dedicated `hasConsoleCommand` already covers this functionality. Could you remove the flag?

Reply:
Done